src/main/java/org/jvnet/hudson/test/WarExploder.java

```diff
@@ -46,6 +48,10 @@
 
     private static final Logger LOGGER = Logger.getLogger(WarExploder.class.getName());
 
+    public static final String JENKINS_WAR_PATH_PROPERTY_NAME = "jth.jenkins-war.path";
+    @CheckForNull
+    private static final String JENKINS_WAR_PATH = System.getProperty(JENKINS_WAR_PATH_PROPERTY_NAME);
```

Comment:
or just `System.getenv("JENKINS_WAR")`?

Reply:
It can be added later if needed.
The biggest problem with that is that the same env name is used by packaging logic in Jenkins: https://github.com/search?q=org%3Ajenkinsci+JENKINS_WAR&type=Code (e.g. Docker packages). If one runs plugin build on a Jenkins with such system environment variables, JTH may pick it and run tests against a wrong WAR. So I prefer explicit definition